VERSION

```diff
@@ -0,0 +1 @@
+v0.4.0
```

Comment:
When this file will be update? Can we update it in the release flow or something else?

Reply:
Yes, we should update this file every time a new version is released

Reply:
Then, please read the release documentation, I think we don't need to setting this value manually. Can we just using environment rather than release file about this value?

Reply:
I checked `release.sh`, from which I can see that the version is controlled through envs. We can compare these two methods:

**use files:**
+ Version information is visible in the version control system, and the change history of the version information can be tracked, including when, by whom and why changes were made.
+ Version information is clearly visible in the project code, making it easy to understand and find.
+ For team members who need to frequently view or modify version information, using files may be more intuitive.

**use env:**
+ Can set different versions for each environment or build without modifying the code.
+ For projects using continuous integration/continuous deployment (CI/CD) tools, environment variables make it easy to set the correct version when building or deploying.

I'm voting to use file and I'd like to hear your and everyone's thoughts.

Reply:
We can use the release page on Github to see all the content you need to view in the file.
If we are using file mode, we still need to modify this file before release, It increases the complexity of maintaining releases. Unless you make the file automatically generated when building the project.

Reply:
Additionally, regardless of the final storage method used, please manage all version-related contents(agent version, Go version number, and Git commit version number) in the same way. Otherwise, other developers or maintenance personnel may become confused.

Reply:
> Additionally, regardless of the final storage method used, please manage all version-related contents(agent version, Go version number, and Git commit version number) in the same way. Otherwise, other developers or maintenance personnel may become confused.

In fact: version, goVersion, and gitCommit records are all determined when the version is released.

Reply:
> We can use the release page on Github to see all the content you need to view in the file. If we are using file mode, we still need to modify this file before release, It increases the complexity of maintaining releases. Unless you make the file automatically generated when building the project.

That seems to make sense and I'll think about it. 
For now, let’s use the env way first!    :)